scripts/poetry.lock

```diff
@@ -0,0 +1,79 @@
+[[package]]
```

Comment:
I wouldn't include a lock file for an internal tool. But no strong opinion on that.

Reply:
On the contrary, I have a strong preference for a lockfile. Because this is a distributed project, making sure that every contributor has the exact same set of dependencies ensures that we're on the same page and not missing any small bugs.

For example I recently learned that Pyarrow v7 updated their LZ4 compression format when writing files, and so LZ4-compressed parquet files written with pyarrow v6 and pyarrow v7 are not the same.